BuildResidentialHPXML/measure.xml

```diff
@@ -92,7 +92,7 @@
     <argument>
       <name>simulation_control_timestep</name>
       <display_name>Simulation Control: Timestep</display_name>
-      <description>Value must be a divisor of 60. If not provided, the OS-HPXML default is used.</description>
+      <description>Value must be a divisor of 60. If not provided, the OS-HPXML default (see [HPXML Simulation Control](https://openstudio-hpxml.readthedocs.io/en/v1.7.0/workflow_inputs.html#hpxml-simulation-control)) is used.</description>
```

Comment:
Hmm. What will this look like in PAT?

Reply:
I'd assume it will just be the whole string.

Reply:
Can you check? We might have to make this a url string and just turn it to a link for the README.

Reply:
Although even with a url string, I wonder if the user can select the text to copy/paste.

Reply:
Confirmed in OSApp that with the "href" approach you can at least right-click and "Copy Link Location".

Reply:
I guess there isn't much we can do for PAT, it would need to be updated to better handle this.